Automattic/woocommerce-payments
# Set price to active when removing subscription product from trash

Fixes #3167

#### Changes proposed in this Pull Request

This PR fixes a bug where, when trying to purchase a subscription product that has been put in the trash and then restored, the customer gets an error and isn't able to make the purchase.

The reason for the bug is that when a subscription product is restored from trash, the subscription product is unarchived on Stripe but the price object remains archived. This PR fixes the issue by setting `active` to `true` in `unarchive_price`.

#### Testing instructions

1. Add a new subscription product, give it a price.
2. "Trash" the subscription product.
3. Go to the Trash and "Restore" the subscription product.
4. As a customer, try to purchase the new subscription product. Ensure that the purchase completes without error.

-------------------


**Post merge**
